src/settings/SupportConfig.cpp

```diff
@@ -0,0 +1,75 @@
+//Copyright (c) 2020 Ultimaker B.V.
+//CuraEngine is released under the terms of the AGPLv3 or higher.
+
+#include "SupportConfig.h"
+
+#include "types/Ratio.h"
+
+namespace cura
+{
+
+bool SupportConfig::LowerLayer(const size_t& layer_no)
```

Comment:
The reference to this parameter is not needed for objects that are smaller than the size of a reference. Also goes for some parameters in the other methods of this class (like the `bool`s).

Reply:
True. I will change that and therefore also remove the `const` since it will be passed by copy and the `const` will be redundant then